File: Exercicio2.c
#include <stdio.h>

int main(void) {
  float raio, area, pi;
  pi = 3.14159;
  
  scanf("%f", &raio);
  area = (pi*raio*raio);
  
  printf("A = %.4f",area);
  return 0;
}

File: Exercicio2.cs
using System;

class Exercicio2 {
  public static void Main (string[] args) {
    double raio, area;
    double pi = 3.14159;
    
    raio = int.Parse(Console.ReadLine());
    area = (pi*raio*raio);
    
  Console.WriteLine ($"{area:F4}");
  }
}
